samples/processing/my-processing-job.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: ProcessingJob
+metadata:
+  name: <YOUR JOB NAME>
+spec:
+  processingJobName: <YOUR JOB NAME> 
+  # Name that will appear in SageMaker console
+  roleARN: <YOUR SAGEMAKER ROLE ARN> 
+  # A role with SageMaker and S3 access
+  # example arn:aws:iam::1234567890:role/service-role/AmazonSageMaker-ExecutionRole
+  appSpecification:
+    imageURI: 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.5.0-cpu-py36-ubuntu16.04
```

Comment:
this is a DLC image not sagemaker 1-p
https://github.com/aws/deep-learning-containers/blob/master/available_images.md